### ISSUE TITLE
Incremental builds

### ISSUE THREAD

**Issue by ekeuus** [a7ul/esbuild-node-tsc#21]
Would it be possible to extend this to have incremental builds using nodemon? Our full dev build using etsc takes about 15s and if incremental could improve it somehow, I'd be willing to implement it. I currently see no such flag is in the config.

**Comment by a7ul** [comment]
Yep sounds good. 
Can you give it a try ?

**Comment by ekeuus** [comment]
I started work on it a bit and actually have a first working version here - https://github.com/fizure/esbuild-node-tsc/commit/0e3d4075431e4be5bb821652b252678c3a8294d6. The problem is, however that we'd still need to restart the script somehow so I am not sure if the complexity is worth it.

**Why it is not worth it** - when I added more logging in the code I found that my combination of basedir '.' and filePatterns: ["**/*.json"] ended up also copying assets from node_modules, which took at least 5 seconds (up to 20). The build itself only took some hundred ms. When only assets from the src folder are copied, the build time is around 2 sec on a very large project so I am not sure if the watch mode, proposed in the commit above, adds value.
